bedrock/firefox/redirects.py

```diff
@@ -692,9 +690,6 @@ def mobile_app(request, *args, **kwargs):
     redirect(r"^/firefox/?$", f"{FXC}/"),
 )
 
-if settings.ENABLE_FIREFOX_COM_REDIRECTS is True:
-    redirectpatterns = (
-        bedrock_redirectpatterns + springfield_redirectpatterns
-    )  # bedrock redirects first, to keep tests, happy, then off to springfield, if relevant
-else:
-    redirectpatterns = bedrock_redirectpatterns
+redirectpatterns = (
+    bedrock_redirectpatterns + springfield_redirectpatterns
+)  # bedrock redirects first, to keep tests, happy, then off to springfield, if relevant
```

Comment:
Whoops - fixing the typo I made!
```suggestion
)  # bedrock redirects first, to keep tests happy, then off to springfield, if relevant
```